src/commands/add.ts

```diff
@@ -0,0 +1,147 @@
+/* eslint-disable no-case-declarations */
+import { getModels } from "../services/app/app.js";
+import {
+  CREATE_GLOBAL_ACTION_MUTATION,
+  CREATE_MODEL_ACTION_MUTATION,
+  CREATE_MODEL_MUTATION,
+  CREATE_ROUTE_MUTATION,
+} from "../services/app/edit/operation.js";
+import { ArgError } from "../services/command/arg.js";
+import type { Command, Usage } from "../services/command/command.js";
+import type { Context } from "../services/command/context.js";
+import { UnknownDirectoryError } from "../services/filesync/error.js";
+import { FileSync } from "../services/filesync/filesync.js";
+import { SyncJson, SyncJsonArgs, loadSyncJsonDirectory } from "../services/filesync/sync-json.js";
+import { println } from "../services/output/print.js";
+import { sprint } from "../services/output/sprint.js";
+import { sortBySimilar } from "../services/util/collection.js";
+import type { DevArgs } from "./dev.js";
+
+export type AddArgs = typeof args;
+
+export const args = {
+  ...SyncJsonArgs,
+  "--model": { type: String, alias: "-m" },
+};
+
+export const usage: Usage = () => {
+  return sprint`
+    Add a new route, model, action.
+
+    {bold USAGE}
+
+      ggt add model [NAME]
+      ggt add action [NAME]
+      ggt add action -m [MODEL] [NAME]
+      ggt add route [METHOD] [NAME]
```

Comment:
CLI's usually differentiate between optional and required arguments using `[OPTIONAL]` vs `<REQUIRED>`.

How do you feel about this instead?

```suggestion
      ggt add model <MODEL>
      ggt add action <ACTION> [--model=<model>]
      ggt add route <METHOD> <NAME>
```